src/codegen/pythoncg.cpp

```diff
@@ -102,7 +105,7 @@ wxString PythonTemplateParser::ValueToCode( PropertyType type, wxString value )
 		}
 	case PT_WXPARENT_CP:
 	{
-		result = value + wxT(".GetPane()");
+		result = wxT("self.")  + value + wxT(".GetPane()");
```

Comment:
What are you trying to fix with this? I don't think this is correct, i did a quick test with the current version and the `self.` gets added in the proper places without your change, with your change this should result in `self.self.` in the places.

Reply:
The problem is when elements are inserted into the collapsePane, and it is necessary to call GetPane ()

For example: With original trunk a have a frame:
![Frame](https://user-images.githubusercontent.com/51421291/59330606-37f09a00-8cf2-11e9-8e80-b39ac89a6845.jpg)

The program generate a code:
![fail_collapsePane](https://user-images.githubusercontent.com/51421291/59330833-d67cfb00-8cf2-11e9-963f-14a692c1b52f.jpg)

With my coding:

![fail_collapsePane](https://user-images.githubusercontent.com/51421291/59330981-2065e100-8cf3-11e9-9f1c-6b0632137b97.jpg)

Insert .self when GetPane is called

Reply:
I still can't reproduce your issue, i have rebuild your layout and this is the generated code:

![collapse](https://user-images.githubusercontent.com/29261375/59376961-7cf7e900-8d51-11e9-9b17-12d7b0f8f833.png)

This is made by wxFB 48f2f0f, which revision are you using?

Reply:
last revision 0e591aaa7a98ca756359bd5dd8cd2cfa9a7d1140
Compile in Windows 10 64 bits with MSYS32 with gcc 9.1.0

Reply:
Ah i see, thats the reason, my revision is too old, it doesn't contain the changes to the code generation that have been made by myself 😄. I can reproduce the issue now and unfortunately not only Python is affected but also PHP and Lua, but for some reason in Lua the error doesn't manifest. In contrast to the wxStaticBoxSizer case the path to the parent wx element needs to get prepended.

This fix is kind of unrelated to the added control, can you please remove it from this PR and file it as a separate one, plus maybe fixing the other two languages as well?

Reply:
ok